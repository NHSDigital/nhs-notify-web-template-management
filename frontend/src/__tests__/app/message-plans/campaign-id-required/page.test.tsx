import { render } from '@testing-library/react';
import MessagePlanCampaignIdRequiredPage from '@app/message-plans/campaign-id-required/page';

test('matches snapshot', () => {
  const container = render(MessagePlanCampaignIdRequiredPage());

  expect(container.asFragment()).toMatchSnapshot();
});
