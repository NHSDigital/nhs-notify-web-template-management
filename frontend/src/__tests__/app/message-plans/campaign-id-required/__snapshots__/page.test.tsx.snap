// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <main
    class="nhsuk-main-wrapper"
    id="maincontent"
    role="main"
  >
    <div
      class="nhsuk-grid-row"
      data-testid="page-content-wrapper"
    >
      <div
        class="nhsuk-grid-column-two-thirds"
      >
        <h1
          class="nhsuk-heading-xl"
          data-testid="page-heading"
        >
          You cannot create message plans yet
        </h1>
        <p>
          To get access, contact your onboarding manager and give them this error message:
        </p>
        <div
          class="nhsuk-inset-text"
        >
          <p>
            Account needs a campaign ID
          </p>
        </div>
        <p>
          <a
            data-testid="back-link"
            href="/message-plans"
          >
            Go back
          </a>
        </p>
      </div>
    </div>
  </main>
</DocumentFragment>
`;
